README.adoc

```diff
@@ -0,0 +1,265 @@
+// Copyright 2019-2021 The Khronos Group, Inc.
+// SPDX-License-Identifier: CC-BY-4.0
+
+= Vulkan^®^ Guide
+:regtitle: pass:q,r[^®^]
+The Khronos{regtitle} Vulkan Working Group
+:data-uri:
```

Comment:
is there a reason for the ToC in the `README.adoc` since there is already one in the `Guide.adoc`?

Reply:
Removed it. In the github viewer it's useless as the ToC is inline with the rest of the document which is itself basically a ToC. In a different renderer like Asciidoctor.js online preview extension, you get a nice sidebar ToC, but that's not the usual use case.